Header header
geometry_msgs/PoseStamped poseTip2
geometry_msgs/PoseStamped poseLink6
geometry_msgs/PoseStamped poseTip1
